cms/static/karma_cms.conf.js

```diff
@@ -30,6 +30,7 @@ var options = {
         {pattern: 'cms/**/*spec.js'},
         {pattern: 'coffee/spec/**/*spec.js'},
         {pattern: 'js/certificates/spec/**/*spec.js'},
+        {pattern: 'js/programs/test/specs/*spec.js'},
```

Comment:
I just tried to remove this line, and then the files associated with program unit test for jasmine stopped working. It complaints that program test spec files are failed to load with 404. I think we do need this line.